load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -1379,19 +1411,83 @@ func (s *relaysSuite) querySharedParams(queryNodeRPCURL string) {
 	s.sharedParams = sharedParams
 }
 
+// queryAppParams queries the current on-chain application module parameters for use
+// over the duration of the test.
+func (s *relaysSuite) queryAppParams(queryNodeRPCURL string) {
+	s.Helper()
+
+	deps := depinject.Supply(s.txContext.GetClientCtx())
+
+	blockQueryClient, err := sdkclient.NewClientFromNode(queryNodeRPCURL)
+	require.NoError(s, err)
+	deps = depinject.Configs(deps, depinject.Supply(blockQueryClient))
+
+	appQueryclient, err := query.NewApplicationQuerier(deps)
+	require.NoError(s, err)
+
+	appParams, err := appQueryclient.GetParams(s.ctx)
+	require.NoError(s, err)
+
+	s.appParams = appParams
+}
+
+// queryProofParams queries the current on-chain proof module parameters for use
+// over the duration of the test.
+func (s *relaysSuite) queryProofParams(queryNodeRPCURL string) {
+	s.Helper()
+
+	deps := depinject.Supply(s.txContext.GetClientCtx())
+
+	blockQueryClient, err := sdkclient.NewClientFromNode(queryNodeRPCURL)
+	require.NoError(s, err)
+	deps = depinject.Configs(deps, depinject.Supply(blockQueryClient))
+
+	proofQueryclient, err := query.NewProofQuerier(deps)
+	require.NoError(s, err)
+
+	params, err := proofQueryclient.GetParams(s.ctx)
+	require.NoError(s, err)
+
+	proofParams, ok := params.(*prooftypes.Params)
+	require.True(s, ok)
+
+	s.proofParams = proofParams
+}
+
+// queryTokenomicsParams queries the current on-chain tokenomics module parameters for use
+// over the duration of the test.
+func (s *relaysSuite) queryTokenomicsParams(queryNodeRPCURL string) {
+	s.Helper()
+
+	deps := depinject.Supply(s.txContext.GetClientCtx())
+
+	blockQueryClient, err := sdkclient.NewClientFromNode(queryNodeRPCURL)
+	require.NoError(s, err)
+	deps = depinject.Configs(deps, depinject.Supply(blockQueryClient))
+
+	var clientConn *grpc.ClientConn
```

Comment:
Why is querying tokenomic params differen than proof and app params?

Reply:
The reason we're using the auto-generated query client is that we don't have the need to implement a tokenomics client querier for now since offchain packages don't need it (yet).

Added a `TODO_TECHDEBT` to use the tokenomics querier once implemented.